### PR TITLE
Netlify script: Build edge & stable docs, as well as the website

### PR DESCRIPTION
**Pull request by ewels**
In the build script for Netlify, don't stop at just building the main website. Clone the Nextflow repo and build the docs for the latest stable / edge releases too.

If this works then deployment previews will also include docs, and we can potentially be closer to automated docs deployment on release 😬

### CODE REVIEW

File: netlify_preview_build.sh [hunk 1]
```diff
@@ -29,3 +33,39 @@ wget --quiet https://github.com/jbake-org/jbake/releases/download/v$jbake_versio
...
+# Fetch the Nextflow repo
+cd ../
+gh repo clone nextflow-io/nextflow
+cd nextflow/docs/
+
+# Find the latest stable and edge releases
+STABLE_TAG=$(gh release list --exclude-drafts -L 10 | grep -v edge | head -n 1 | cut -f3)
+EDGE_TAG=$(gh release list --exclude-drafts -L 10 | grep edge | head -n 1 | cut -f3)
+echo "Latest stable release: $STABLE_TAG"
+echo "Latest stable release: $EDGE_TAG"
```
Comment (bentsherman) [inline netlify_preview_build.sh:55]:
```suggestion
echo "Latest edge release: $EDGE_TAG"
```

Comment (ewels) [reply netlify_preview_build.sh:55]: Good catch, thanks!

**Review by bentsherman — APPROVED** [review]
Looks great to me!

**Comment by ewels** [comment]
NB: If merged and if we want the Netlify site to be up to date, we also need to add some GitHub actions magic to the main Nextflow repo to trigger a deployment build whenever there's a new release. Can do that later though, shouldn't be too difficult..

**Comment by ewels** [comment]
Closing in favour of https://github.com/nextflow-io/website/pull/174 - there it's in the main build scripts and not netlify, so should work for AWS s3 deployment as well as Netlify previews.

**Pull request closed without merging**